### ISSUE TITLE
Task lists for Weeks of 1/25 and 2/1

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#376]
Tasks for @ebachoura and @eileenchoe: 
- [x] @ebachoura and @eileenchoe need to schedule safety training with @kdahlquist #377 
- [ ] #326 @ebachoura and @eileenchoe update state diagram with feature to show/hide all weights, pending instructions from @dondi to deal with complexity of diagram
- [ ] #361 update documentation as they suggested in their comment
- [ ] work on "syntactic" tests for SIF and graphml input files. #362 and #363 with @mihirsamdarshi 

Tasks for @NAnguiano and @yshin4 
- [x] work on finite/infinite bounding box #345 (rearranging partners so that they can work together on Tuesdays)  @NAnguiano should show @yshin4 what she is working on and we'll go from there in terms of how to assign specific tasks. 
- [x] need status update on #343.

Tasks for @anuvarsh and @mihirsamdarshi:
- [x] Southern California Conference for Systems Biology poster #360 

Tasks for @anuvarsh:
- [ ] #285 has cosmetic and redraw issues (needs to talk to @dondi )
- [x] #368 outline for BIOL 598
- [ ] #264 user defined normalization factor

Tasks for everyone
- [x] abstracts for LMU Symposium #358 
- [ ] #382, begin using forks and pull requests to manage updates to the repository
- [x] #383, everyone do a clean room install of libraries, agree upon versions and update wiki, update package.json

Tasks for @dondi 
- [x] give guidance on #326 
- [x] #375 set up Zenodo (if time)
- [x] #300 review requested
- [ ] #388 multiple servers for testing

Tasks for @kdahlquist
- [ ] upload poster to SlideShare, news item, publication item
- [x] show @NAnguiano samples of scrolling/zooming buttons

**Comment by kdahlquist** [comment]
I updated the task list as per what we discussed in the meeting.

**Comment by dondi** [comment]
We can check off the #377 task once all confirmations for the proposed date/time are in.

**Comment by dondi** [comment]
@dondi added a description of how to proceed with #326 in that issue's comment thread.

**Comment by dondi** [comment]
#361 led to #383 and #383 is now the top priority.

**Comment by dondi** [comment]
@dondi walked @eileenchoe, @ebachoura, and @mihirsamdarshi through the syntactic error tasks.

**Comment by dondi** [comment]
@NAnguiano will look at https://github.com/d3/d3/wiki/Gallery to target potential alternative visualizations for GRNs and perhaps discuss the best options with @kdahlquist

**Comment by dondi** [comment]
For #345 @NAnguiano has a proposed combination of scroll/zoom functionality which is ready for review. She will make this available on beta and signal when we can have a look.

**Comment by dondi** [comment]
@dondi will look for options to address the weights refresh issue with @anuvarsh (#285); she is halfway done with #368; and #264 specification has been settled and implementation will proceed after #285.

**Comment by dondi** [comment]
#358 will be worked on during the 2/1 work session; thematically @anuvarsh and @NAnguiano focus on visualization changes while the group poster reflects data interoperability and integrity improvements.

**Comment by dondi** [comment]
@dondi has updated #382 with details on how this structure will work. Let's process this for a bit then have everyone sign off once we've settled the new work flow and structure.

**Comment by dondi** [comment]
Updates to #375 and #300 have been made.

**Comment by kdahlquist** [comment]
#389 is now a task for @kdahlquist, @dondi, and @NAnguiano .

**Comment by kdahlquist** [comment]
All open tasks have been moved over to new issue #391 .
